### ISSUE TITLE
Comment storage

### ISSUE THREAD

**Issue by saulshanabrook** [jupyterlab/jupyterlab-commenting#34]
Questions:

* Where do we store comments? 
* Do we store them in multiple places?
  * These places are in a `comments.json` file or inside a notebook. 
  * Do we have a way to serialize/export the comments from remote storage to this local storage?

**Comment by ktaletsk** [comment]
Would it be possuble to store the comments in the notebook metadata? That will make the comments portable

**Comment by saulshanabrook** [comment]
@ktaletsk Yeah, we are currently experimenting with this, storing them in the notebook metadata.
